### DISCUSSION TITLE
Person detection triggering outside zone

### DISCUSSION THREAD

**Discussion by jsbrich** [blakeblackshear/frigate#8035] [General]
I have set up several zones and object detection in my setup.  All, but one, are working as expected.  For some reason, Person in the All Driveway zone triggers even with people outside of the zone.  I've attached an example (person in the upper center), the zone markup, and the relevant section from the config.  Car in the Driveway Entrace zone works fine, not picking up cars driving by on the street.  Any ideas?
![Screenshot 2023-10-02 135005](https://github.com/blakeblackshear/frigate/assets/63560376/715dab81-ae32-430b-9ba3-bbfa910c4fe9)
![snapshot](https://github.com/blakeblackshear/frigate/assets/63560376/a125cab8-046a-4454-bdd2-ee6f2f17bab2)
![Screenshot 2023-10-02 134755](https://github.com/blakeblackshear/frigate/assets/63560376/db73c894-11ae-44c1-b951-2a2dfbe3735f)
![Screenshot 2023-10-02 133942](https://github.com/blakeblackshear/frigate/assets/63560376/918b262c-c3f8-4a40-a684-7c123253d393)

**Comment by NickM-27** [comment]
It is important to note that the snapshot is not necessarily the moment that the person was detected in the zone. In 0.12 frigate's tracker is less sophisticated and might be confusing a false positive in the zone to be the same person as the one that is across the street.

In Frigate 0.13 the tracker is better and there is a recordings timeline feature which will show you the exact moment (and the detection box) that the object was considered entering the zone

**Reply by jsbrich** [reply]
The recorded clip (as well as my full recordings) show the person never crossed the street, so wouldn't have been in the zone.  I didn't want to include the clips for privacy and size reasons.

**Reply by NickM-27** [reply]
Like I said, the model likely detected something that was not a person as a person in the zone, and frigate assumed that "not person" was the same person that is walking across the street.

**Comment by jsbrich** [comment]
> In Frigate 0.13 the tracker is better and there is a recordings timeline feature which will show you the exact moment (and the detection box) that the object was considered entering the zone

I see 0.13 is still listed as beta, how close to "stable" is it?

**Reply by NickM-27** [reply]
there are no major or medium size issues that are known, main things are docs updates and UI fixes that are currently being done. If you did want to run it, I'd highly suggest reading the breaking changes and the release notes in general before updating

**Reply by jsbrich** [reply]
Probably stupid question, but how do I install the 5.0 HA Integration?  I'm currently using the 4.0.0, but don't see any mechanism in HACS to tell it to upgrade to the beta?

**Reply by jsbrich** [reply]
Figured it out, have to remove the current and then there is an option to show beta versions when you re-add

**Reply by NickM-27** [reply]
HACS -> three dots on frigate integration -> download other version -> check allow beta versions -> choose beta version
